lib/swaps/Swaps.ts

```diff
@@ -96,6 +101,16 @@ class Swaps extends EventEmitter {
     return { takerAmount, makerAmount };
   }
 
+  public init = async () => {
+    // Load Swaps from data base
+    const promises: PromiseLike<any> = this.repository.getSwaps();
+    const result = await Promise.resolve(promises);
+    result.map((deal: SwapDeal) => {
```

Comment:
Is `map` the right function here? I would think we'd want `forEach`. Either way this would be moot if we go with my suggestion to pull back an array of hashes from the database rather than the full instances.